### PR TITLE
[DH-822] Adds kind, company and contact filters to interaction.

### PR DESCRIPTION
**Pull request by elcct**

### CODE REVIEW

File: datahub/search/interaction/tests/test_views.py [hunk 1]
```diff
@@ -177,6 +178,51 @@ def test_search_original_query(self, interactions, term):
             'modified_on': interaction.modified_on.isoformat(),
         }]
 
+    def test_filter_by_kind(self, setup_es):
+        """Tests filtering interaction by kind."""
+        interaction = InteractionFactory(
+            kind=Interaction.KINDS.service_delivery,
```
Comment (reupen) [inline datahub/search/interaction/tests/test_views.py]: I think we need some interactions that shouldn't match in here as well (same for the other tests).

**Comment by ztolley** [comment]
Does it add investment project? This filter is also required so we can list interactions for an investment project.

**Comment by elcct** [comment]
@ztolley it doesn't add investment project as filter, because it is already implemented.

File: datahub/search/interaction/tests/test_views.py (hunk 1 above)
Comment (elcct) [reply datahub/search/interaction/tests/test_views.py]: I now added more Interactions that don't match.

File: datahub/search/interaction/views.py [hunk 2]
```diff
@@ -12,7 +12,10 @@ class SearchInteractionParams:
     serializer_class = SearchInteractionSerializer
 
     FILTER_FIELDS = (
+        'kind',
+        'company',
         'company_name',
+        'contact',
         'contact_name',
         'dit_adviser_name',
         'dit_team',
```
Comment (ztolley) [inline datahub/search/interaction/views.py]: i didn't see the investment_project here to assumed it wasn't a supported filter

**Pull request merged**